src/JWT/JWTBuilder/Enums/HeaderNames.cs

```diff
@@ -0,0 +1,18 @@
+using System.ComponentModel;
+
+namespace JWT.JWTBuilder.Enums
+{
+    /// <summary>
+    /// All predefined Headers specified by RFC. (https://tools.ietf.org/html/rfc7519)
+    /// Last update 31.10.2017
```

Comment:
Do we need the last update timestamp?

Reply:
I don't how often this list will upadte.